kola/harness.go

```diff
@@ -107,6 +108,18 @@ func filterTests(tests map[string]*register.Test, pattern, platform string) (map
 			continue
 		}
 
+		var minVersion *semver.Version
+		if t.MinVersion == "" {
```

Comment:
Looking back on this, I think I went through this janky step at the time because its slightly more ergonomic for a test writer to specify `MinVersion: "v1.1.1"` then `Min Version: semver.Must(semver.NewVersion("v1.1.1")`. 

Its certainly more simple in the code to use a `semver.Version` though. Do you still feel strongly either way?